content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository.md

```diff
@@ -68,7 +68,7 @@ If the branch you want to delete is associated with an open pull request, you mu
 
 {% data reusables.repositories.navigate-to-repo %}
 {% data reusables.repositories.navigate-to-branches %}
-1. Next branch that you want to delete, click {% octicon "trash" aria-label="Delete BRANCH" %}.
+1. Click {% octicon "trash" aria-label="Delete BRANCH" %} next to the branch that you want to delete.
```

Comment:
LGTM, following our content model that mentions the location on the page first, and then mentions the action. Also changed the `aria-label` parameter to follow our latest accessibility guidelines (should indicate the icon name, not the action that the icon allows a user to implement.)
```suggestion
1. Next to the branch that you want to delete, click {% octicon "trash" aria-label="The trash icon" %} .
```